python/setup.py

```diff
@@ -109,7 +112,11 @@ def get_cuda_version_from_header(cuda_include_dir):
     cuda_include_dir,
 ]
 
-library_dirs = [get_python_lib(), os.path.join(os.sys.prefix, "lib")]
+library_dirs = [
+    get_python_lib(),
+    os.path.join(os.sys.prefix, "lib"),
+    cuda_lib_dir,
```

Comment:
Just to confirm my understanding. If building in a conda environment this would still use the libraries from the conda environment before the ones in the `/usr/local/cuda` directory, correct?

Reply:
Yup, that's correct. 

`
...The directories are searched in the order in which they are specified on the command line. Directories specified on the command line are searched before the default directories. All -L options apply to all -l options, regardless of the order in which the options appear. -L options do not affect how ld searches for a linker script unless -T option is specified....
`

ref: https://sourceware.org/binutils/docs-2.21/ld/Options.html#Options